components/snackbar/snackbar.css

```diff
@@ -0,0 +1,15 @@
+@import "../../shared.css";
+
+.snackbar{
+    background-color: var(--orange-500);
+    color: var(--secondary);
+    display: inline-block;
+    border-radius: 0.5rem;
```

Comment:
suggestion:use px for border radius

Reply:
I'm not agree with this . read this : - [https://stackoverflow.com/questions/23991054/does-rem-make-sense-on-border-radius](url)